### ISSUE TITLE
Don't reset the accounts when changing bank's global status

### ISSUE THREAD

**Issue by flaree** [Cog-Creators/Red-DiscordBot#4068]
# Feature request

#### Select the type of feature you are requesting:

- [X] Cog
- [ ] Command
- [ ] API functionality

#### Describe your requested feature

I feel that the wiping of economy when switching has no real impact. I'd like to be able to go back should I not like it and resume from where the leaderboard was at at the time I switched

**Comment by Drapersniper** [comment]
Im in favour of this however...

I can see 2 problems here.

1 - storage size (with recent improvements to json driver this is less of an issue)
2 - Would it be confusing to the user if their currency is reset to what it was on last change?
  say they has 1M in global and 50k in Server, then you toggle from global to server their balance will go from 1M to 50k (or 0 in most cases) without any sort of information to the user I think that could be confusing.

**Comment by flaree** [comment]
I mean its kinda confusing to users right now, they look and see a server balance of X then the owner decides to go global and boom everything's now 0.

**Comment by Drapersniper** [comment]
touche, guess that your suggestion here doesnt make anything any worse.
I've upvote this proposal, there no valid reason I currently see for wiping the bank on a type change.

**Comment by Jackenmen** [comment]
I don't really agree with this, I think it's gonna be more confusing if the accounts go back to whatever they had in the other state after you switch bank's global status. IMO, you shouldn't carelessly be switching the global status of the bank in the first place.

**Comment by flaree** [comment]
I feel they should be allowed to switch between without losing data. When I first started using Red I liked to play around with stuff and find out  what preferences I liked. To do that I needed to experiment and the losing of the data is a bad way to do it. Atleast having the user choose to clear it will be a better way.

**Comment by Drapersniper** [comment]
As a bot owner I did have instances where I wanted the currency changed without wiping it though. So just advocating for that.

**Comment by Drapersniper** [comment]
We could just add a keyword that option that resets without wiping.  And keep the current behaviour the same.

**Comment by yamikaitou** [comment]
My opinion would be to combine the accounts if switching from Guild to Global. But if you switch from Global to Guild, reset it.

**Comment by Drapersniper** [comment]
i erm i dont likee the idea of combining ...
some servers could have a completely out of wack currency prior to conversion.

**Comment by yamikaitou** [comment]
True, didn't consider that. Then I agree with the optional flag/command to either keep or wipe it (I'd say default to wipe)
